Makefile

```diff
@@ -3,7 +3,7 @@ COMMAND=docker-compose -f docker/docker-compose.yml
 install: set-env up composer
 
 set-env:
-	printf "UID=`id -u`\nGID=`id -g`\nIDE_SERVER_NAME=oathservice" > .env
+	printf "WEB_PORT=80\nDB_PORT=3306\nUID=`id -u`\nGID=`id -g`\nIDE_SERVER_NAME=oathservice" > .env
```

Comment:
I've added the configuration options to the readme.

The `set-env` should only be run once to have the default settings written to `.env`. The .env file itself isn't in versioning so feel free to change the options to your needs.

Reply:
Alright, wasn't aware of that, this would work perfectly I suppose. The only caveat is that when running the set-env on a successive run, you would override your previously configured values.

In a perfect world, this command would trigger a confirmation or create a backup of the previous .env file. For now, let's stick to what we have!